src/storage/db_interface_frontend.py

```diff
@@ -249,3 +249,33 @@ def rest_get_file_object_uids(self, offset, limit, query=None):
     def rest_get_object_uids(database, offset, limit, query):
         uid_cursor = database.find(query, {'_id': 1}).skip(offset).limit(limit)
         return [result['_id'] for result in uid_cursor]
+
+    def find_missing_files(self):
+        uids_in_db = set()
```

Comment:
This will in a line 260 include all uids stored in the database. Correct?

Reply:
that is correct, yes